bloom/commands/git/import_upstream.py

```diff
@@ -38,6 +38,7 @@
 import tarfile
 import tempfile
 
+from io import open
```

Comment:
I don't have authority here so I'll leave it up to @wjwwood to decide. I'd personally lean toward using `io.open` everywhere to avoid having to think about it. To keep it explicit, I'd call explicitly `io.open` so that we dont need to check the beginning of the file to know if it has been imported from `io` or not